File: app/src/main/java/com/example/empty/MainActivity.java
package com.example.empty;

import android.os.Bundle;
import android.view.View;

import androidx.appcompat.app.AppCompatActivity;

import io.flutter.embedding.android.FlutterActivity;


public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        findViewById(R.id.btn).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(
                        FlutterActivity
                                .withNewEngine()
                                .build(MainActivity.this)
                );
            }
        });
    }
}

File: app/src/main/java/com/example/empty/MainActivity.kt
package com.example.empty

import android.app.Activity
import android.os.Bundle
import com.example.library.LibViewModel

class MainActivity : Activity() {

    lateinit var viewModel: LibViewModel
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

    }
}